www/include/home/customViews/triggers/loadServiceFromHost.php

```diff
@@ -62,27 +62,29 @@
 $xml->startElement('response');
 try {
     $xml->startElement('options');
-    if ($data) {
+    if ($hostId) {
```

Comment:
Logically incorrect, what's happen if id = 0 ?
```suggestion
    if ($hostId !== false) {
```

Reply:
IMHO -> if id = 0, it will process the query for nothing, as id = 0 is not a consistent value currently

Reply:
Ok, but in this case a negative value is not an acceptable.
So i prefer using a double check with ` if ($hostId !== false && $hostId > 0)`